libs/features/richresults.js

```diff
@@ -33,12 +33,23 @@ function getRichResultsForSiteSearchBox(getMetadata) {
   };
 }
 
+function getRichResultsForOrgLogo(getMetadata) {
+  return {
+    '@context': 'https://schema.org',
+    '@type': 'Organization',
+    url: getMetadata('orgurl'),
+    logo: getMetadata('orglogo'),
+  };
+}
+
 function getRichResults(type, getMetadata) {
   switch (type) {
     case 'NewsArticle':
       return getRichResultsForNewsArticle(getMetadata);
     case 'SiteSearchBox':
       return getRichResultsForSiteSearchBox(getMetadata);
+    case 'organization':
```

Comment:
nit: Uppercase to match Schema.org names but I realize this is passed in from `<meta name="richresults" content="organization">` which may already exist.

Reply:
Yea I agree. Word will auto capitalize this too it seems, so good call thanks :)